-------------------------------------------------------------------------------
Test set: models.dao.Sql2oShopDaoTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.325 sec <<< FAILURE! - in models.dao.Sql2oShopDaoTest
models.dao.Sql2oShopDaoTest  Time elapsed: 1.323 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "41.84.152.6", user "dymqwdieiyxasa", database "d7dmvoatoj65", SSL off
	at models.dao.Sql2oShopDaoTest.setUp(Sql2oShopDaoTest.java:24)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "41.84.152.6", user "dymqwdieiyxasa", database "d7dmvoatoj65", SSL off
	at models.dao.Sql2oShopDaoTest.setUp(Sql2oShopDaoTest.java:24)

models.dao.Sql2oShopDaoTest  Time elapsed: 1.325 sec  <<< ERROR!
java.lang.NullPointerException
	at models.dao.Sql2oShopDaoTest.shutDown(Sql2oShopDaoTest.java:36)

